index.js

```diff
@@ -1,11 +1,24 @@
-const { app, BrowserWindow, dialog, shell, ipcMain } = require('electron');
+const {
+	app,
+	BrowserWindow,
+	dialog,
+	shell,
+	ipcMain,
+	globalShortcut,
+} = require('electron');
 const windowStateKeeper = require('electron-window-state');
 const {
 	hasScreenCapturePermission,
 	openSystemPreferences,
 } = require('mac-screen-capture-permissions');
 const { autoUpdater } = require('electron-updater');
 
+const reloadShortCuts = [
+	'CommandOrControl+R',
+	'CommandOrControl+Shift+R',
+	'F5',
```

Comment:
I don't think F5 triggers a reload inside the official Discord app

Reply:
Fixed in commit https://github.com/yannhodiesne/Discord-M1/pull/17/commits/6cd53a4b4f27ffd687d14444fd29d4f6a32cc2a2